CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# dbt_zendesk v0.17.0
+
+## Breaking Changes (Full refresh required after upgrading)
+- Incremental models have had the `partition_by` logic adjusted to include a granularity of a month. This change should only impact BigQuery warehouses and was applied to avoid the common `too many partitions` error some users have experienced do to over partitioning by day. Therefore, adjusting the partition to a month granularity will increase the partition windows and allow for more performant querying and incremental loads. This change was applied to the following models:
```

Comment:
```suggestion
- Incremental models running on BigQuery have had the `partition_by` logic adjusted to include a granularity of a month. This change only impacts BigQuery warehouses and was applied to avoid the common `too many partitions` error some users have experienced when partitioning by day. Therefore, adjusting the partition to a month granularity will decrease the number of partitions created and allow for more performant querying and incremental loads. This change was applied to the following models:
```
Just a suggestion in wording here. I thought we should emphasize that the number of partitions is decreased since the error is that there were too many.